Soumyo/ec2 routes (#350)
inspec/inspec-aws: docs/resources/aws_route_table.md

# aws\_route\_table

> Use the `aws_route_table` InSpec audit resource to test properties of a single Route Table. A route table contains a set of rules, called routes, that are used to determine where network traffic is directed. This particular resource also test properties of a Route.

Reviewer: ```suggestion
Use the `aws_route_table` InSpec audit resource to test the properties of a single route or route table. A route table contains a set of rules, called routes, that are used to determine where network traffic is directed.
```

      its('propagating_vgws') { should be_empty }
    end

> ##### Confirm that the route table has expected destination_cidr_block of the route

Reviewer: ```suggestion
##### Confirm that the route table has the expected destination_cidr_block of the route
```
